frontend/src/components/config-details.ts

```diff
@@ -380,14 +408,14 @@ export class ConfigDetails extends LiteElement {
   private renderAnchorLink(id: string) {
```

Comment:
Do we want to fix the anchor links? Otherwise, let's remove `renderAnchorLink` and all references

Reply:
I think get rid of them?  We currently only have them on the workflow settings and not the crawl settings page which isn't great for consistency and I don't really see them being used much given that this has been broken for a while presumably and hasn't received much use?  No _super_ strong feelings either way tho?

Reply:
Yeah in that case let's this method and references to it!